html/js/product-multilingual.js

```diff
@@ -494,8 +494,111 @@ function update_display(imagefield, first_display) {
 	$(document).foundation('equalizer', 'reflow');
 }
 
+function initializeTagifyInputs() {
+  const inputs = document.querySelectorAll('input.tagify-me');
+  for (let i = 0; i < inputs.length; ++i) {
+    initializeTagifyInput(inputs[i]);
+  }
+}
+
+const maximumRecentEntriesPerTag = 3;
+function initializeTagifyInput(el) {
+  const input = new Tagify(el, {
+    autocomplete: true,
+    whitelist: get_recents(el.id) || [],
+    dropdown: {
+      enabled: 0
+    },
+  });
+
+  let abortController;
+  input.on('input', function(e) {
+    const value = e.detail.value;
+    input.settings.whitelist = []; // reset the whitelist
+
+    if (el.dataset.autocomplete && el.dataset.autocomplete !== "") {
+      // https://developer.mozilla.org/en-US/docs/Web/API/AbortController/abort
+      if (abortController) {
+        abortController.abort();
+      }
+
+      abortController = new AbortController();
+
+      fetch(el.dataset.autocomplete + 'term=' + value, {signal: abortController.signal}).
+        then((RES) => RES.json()).
+        then(function(whitelist) {
+          input.settings.whitelist = whitelist;
+          input.dropdown.show.call(input, value); // render the suggestions dropdown
+      });
+    }
+  });
+
+  input.on('add', function(e) {
+    let obj;
+
+    try {
+      obj = JSON.parse(window.localStorage.getItem("po_last_tags"));
+    } catch(e) {
+      if(e.name == "NS_ERROR_FILE_CORRUPTED") {
+          obj = null;
+      }
+    }
+
+    const tag = e.detail.data.value;
+    if (obj === null) {
+      obj = {};
+      obj[el.id] = [tag];
+    } else if (obj[el.id] === null) {
+      obj[el.id] = [tag];
+    } else {
+      if (obj[el.id].indexOf(tag) != -1) {
+        return;
+      }
+
+      if (obj[el.id].length >= maximumRecentEntriesPerTag) {
+        obj[el.id].pop();
+      }
+
+      obj[el.id].unshift(tag);
+    }
+
+
+    try {
+      window.localStorage.setItem("po_last_tags", JSON.stringify(obj));
+    } catch(e) {
+      if(e.name == "NS_ERROR_FILE_CORRUPTED") {
+      }
+    }
+
+    input.settings.whitelist = obj[el.id]; // reset the whitelist
+  });
+
+  document.getElementById('product_form').addEventListener('submit', function() {
+    el.value = input.value.map((obj) => obj.value).join(',');
+  });
+}
+
+function get_recents(tagfield) {
+  let obj
+    try {
+      obj = JSON.parse(window.localStorage.getItem("po_last_tags"));
+    } catch(e) {
+      if(e.name == "NS_ERROR_FILE_CORRUPTED") {
+        obj = null;
+      }
+    }
+
+  if (obj === null) {
+    return [];
+  }
+
+  return obj[tagfield] === null ? [] : obj[tagfield];
```

Comment:
```suggestion
  return obj[tagfield] == null ? [] : obj[tagfield];
```

Reply:
The current eslint ruleset (based on some standard that I don't remember 🤷‍♂) would complain about the missing third `=`: https://eslint.org/docs/rules/no-eq-null

Reply:
If we want to keep that rule just create an if with === null or === undefined or === ""